network/horizon/api-reference/resources/operations/object/change-trust.mdx

```diff
@@ -45,19 +45,19 @@ See the [`Change Trust` errors](../../../errors/result-codes/operation-specific/
 {
   "_links": {
     "self": {
-      "href": "https://horizon.stellar.org/operations/120192477935251457"
+      "href": "https://horizon-testnet.stellar.org/operations/120192477935251457"
```

Comment:
these all become broken links now cuz those operations/etc. don't exist on testnet :/

Reply:
I know and testnet also gets reset all the time. if testnet doesn't get reset, we can find permanent href to point to